octoprint_mrbeam/static/js/mrbeam.js

```diff
@@ -440,10 +443,17 @@ $(function () {
         };
 
         self._handle_session_expired = function (triggerUrl) {
+            // don't do this during boot time.
+            if (
+                self.isCurtainOpened <= 0 &&
+                self.isCurtainOpened + 10 * 1000 > Date.now()
```

Comment:
is 10 seconds here an assumption for how long the booting takes??

Reply:
I'm not sure the "less than" services any purpose here. please clarify.

Reply:
1. It is the moment the "onCurtainOpened" event fired + 10 seconds. 
2. self.isCurtainOpened is initialised to 0 (I see it as a timestamp). It remains zero until the "onCurtainOpened" event fires, then it is set ti the current timestamp. The "less than" is "just to be on the save side" incase someone initialises or sets it's value to -1.

Reply:
wouldn't it be easier to check if the curtain is still active or displayed?